src/index.js

```diff
@@ -0,0 +1,86 @@
+// import packages
+const inquirer = require("inquirer");
+const questions = require("./utils/questions");
```

Comment:
Let's destructure the import directly here: 

`const {
  getManagerQuestions,
  getContinueQuestion,
  getEngineerQuestions,
  getInternQuestions,
} = require("./utils/questions")
```